src/test/javascript/spec/app/account/register/register.component.spec.ts

```diff
@@ -1,14 +1,19 @@
 import { async, ComponentFixture, fakeAsync, inject, TestBed, tick } from '@angular/core/testing';
 import { FormBuilder } from '@angular/forms';
+
+import { EventPlannerTestModule } from '../../../test.module';
```

Comment:
Was ist das den für ein Modul?

Reply:
da sind allgemeine Test-sachen drinne :P ist nicht neu, nur in anderer Reihenfolge importiert.